RNTester/android/app/build.gradle

```diff
@@ -4,10 +4,9 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-apply plugin: 'com.android.application'
-
-import com.android.build.OutputFile
+plugins {
```

Comment:
File extension looks like not changed.

Reply:
I'm touching only top level Gradle scripts this time.

Reply:
I think we better do it all.

Reply:
it'll require a lot of work, and it's not trivial to convert to Kotlin DSL due to static typing